akka/src/test/scala-2.10/better/files/ActorSystemSupport.scala

```diff
@@ -0,0 +1,10 @@
+package better.files
+
```

Comment:
to allow upgrade of akka to a 2.12 supported version.
newer akka replaced stop with terminate.
2.11/2.12 has the same api but that might change as well.